core/storage/email/gae_models.py

```diff
@@ -184,6 +185,40 @@ def _pre_put_hook(self):
         self.email_hash = self._generate_hash(
             self.recipient_id, self.subject, self.html_body)
 
+    @classmethod
+    def create_multi(cls, send_email_infos):
+        """Creates SentEmailModel entries for multiple emails.
+
+        Args:
+            send_email_infos: list(SendEmailInfo). Each SendEmailInfo object
+                contains the information needed to generate a SentEmailModel.
```

Comment:
```suggestion
                containing the information needed to generate a SentEmailModel.
```

Nit: the tense isn't quite correct here.

Reply:
I think the tense is right?

Reply:
You're right. I was reading this differently while skipping some words apparently. :)